test/support/views/blocks/example_view.ex

```diff
@@ -0,0 +1,8 @@
+defmodule ColonelKurtzTest.Blocks.ExampleView do
+  use Phoenix.HTML
```

Comment:
This is also a bit of a hack. Once we have a proper phoenix app spun up we can change this over to `use TestApp.View, :block_view` or something more realistic.